datahub-web-react/src/app/entity/tag/TagProfile.tsx

```diff
@@ -43,6 +82,24 @@ export default function TagProfile() {
     const { urn } = useParams<TagPageParams>();
     const { loading, error, data } = useGetTagQuery({ variables: { urn } });
     const entityRegistry = useEntityRegistry();
+    const history = useHistory();
+
+    const allSearchResultsByType = useGetAllEntitySearchResults({
+        query: `tags:${data?.tag?.name}`,
+        start: 0,
+        count: 1,
```

Comment:
We want to limit to 1 result here? What if there are 5?

Reply:
The total tells us all we need anyway!